실험 'python.createEnvironment.trigger'이(가) 활성 상태입니다.
실험 'pythonaa'이(가) 활성 상태입니다.
실험 'pythonPromptNewFormatterExt'이(가) 활성 상태입니다.
실험 'pythonPromptNewToolsExt'이(가) 활성 상태입니다.
실험 'pythonTestAdapter'이(가) 활성 상태입니다.
LSP Notebooks experiment is enabled
LSP Notebooks interactive window support is enabled
Python 인터프리터 경로: ./.pyenv/versions/3.9.11/bin/python
Pylance 언어 서버를 시작하는 중입니다.
> conda info --json
> ./.pyenv/versions/3.7.13/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.8.13/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.7.13/envs/3.7.7/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.9.11/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/shims/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./C:UsersUserDesktop/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.7.13/envs/django-envs/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./폴더/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.7.13/envs/django_coplate_env/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.7.13/envs/django_podo_env/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./.pyenv/versions/3.9.11/envs/venv/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /usr/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-9-2 22:26:51.409]: Support for Python 2.7 has been dropped by the Python extension so certain features may not work, upgrade to using Python 3.
> /usr/bin/python ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> /usr/bin/python3 -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
> ./C:UsersUserDesktop새/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-9-2 22:26:51.508]: [Error: Command failed: /home/aa/C:UsersUserDesktop새/bin/python -I /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 1: /home/aa/C:UsersUserDesktop새/bin/python: not found

	at ChildProcess.exithandler (node:child_process:399:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Socket.<anonymous> (node:internal/child_process:451:11)
	at Socket.emit (node:events:526:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 127,
  signal: null,
  cmd: '/home/aa/C:UsersUserDesktop새/bin/python -I /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
> ./C:UsersUserDesktop새/bin/python -I ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py ./.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
[ERROR 2023-9-2 22:26:53.516]: [Error: Command failed: /home/aa/C:UsersUserDesktop새/bin/python -I /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py
/bin/sh: 1: /home/aa/C:UsersUserDesktop새/bin/python: not found

	at ChildProcess.exithandler (node:child_process:399:12)
	at ChildProcess.emit (node:events:526:28)
	at maybeClose (node:internal/child_process:1092:16)
	at Socket.<anonymous> (node:internal/child_process:451:11)
	at Socket.emit (node:events:526:28)
	at Pipe.<anonymous> (node:net:687:12)] {
  killed: false,
  code: 127,
  signal: null,
  cmd: '/home/aa/C:UsersUserDesktop새/bin/python -I /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/get_output_via_markers.py /home/aa/.vscode-server/extensions/ms-python.python-2023.4.1/pythonFiles/interpreterInfo.py'
}
Send text to terminal: pyenv shell 3.9.11
